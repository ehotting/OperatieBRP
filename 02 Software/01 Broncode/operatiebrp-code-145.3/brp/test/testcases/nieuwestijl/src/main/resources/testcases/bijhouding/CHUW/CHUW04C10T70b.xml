<brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens brp:communicatieID="CI_stuurgegevens">
        <brp:zendendePartij>507013</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>88409eeb-1aa5-43fc-8614-43055123a165</brp:referentienummer>
        <brp:tijdstipVerzending>2016-05-11T08:32:03.234+01:00</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:parameters brp:communicatieID="CI_parameters">
        <brp:verwerkingswijze>Bijhouding</brp:verwerkingswijze>
    </brp:parameters>
    <brp:ontbindingHuwelijkInBuitenland brp:objecttype="AdministratieveHandeling" brp:communicatieID="CI_adm_hand">
        <brp:partijCode>507013</brp:partijCode>
        <brp:toelichtingOntlening>Test toelichting op de ontlening</brp:toelichtingOntlening>
        <brp:bronnen>
            <brp:bron brp:objecttype="AdministratieveHandelingBron" brp:communicatieID="CI_ah_bron_1">
                <brp:document brp:objecttype="Document" brp:communicatieID="CI_document_1">
                    <brp:soortNaam>Buitenlandse rechterlijke uitspraak</brp:soortNaam>
                    <brp:partijCode>507013</brp:partijCode>
                </brp:document>
            </brp:bron>
            <brp:bron brp:communicatieID="comid.ontbindingHuwelijkInNederland.bron2" brp:objecttype="AdministratieveHandelingBron">
                <brp:document brp:communicatieID="comid.ontbindingHuwelijkInNederland.bron2.document" brp:objecttype="Document">
                    <brp:soortNaam>Verklaring of kennisgeving naamgebruik</brp:soortNaam>
                    <brp:partijCode>507013</brp:partijCode>
                </brp:document>
            </brp:bron>
        </brp:bronnen>
        <brp:acties>
            <brp:registratieEindeHuwelijk brp:objecttype="Actie" brp:communicatieID="CI_actie_reg_huw_gp">
            <brp:datumAanvangGeldigheid>2016-05-11</brp:datumAanvangGeldigheid>
                <brp:bronnen>
                    <brp:bron brp:objecttype="ActieBron" brp:referentieID="CI_ah_bron_1" brp:communicatieID="CI_actie_bron"/>
                </brp:bronnen>
                <brp:huwelijk brp:objecttype="Relatie" brp:communicatieID="CI_huwelijk" brp:objectSleutel="30010001">
                    <brp:relatie brp:communicatieID="CI_huwelijk_groep_relatie">
                        <brp:redenEindeCode>A</brp:redenEindeCode>
                        <brp:datumEinde>2016-05-11</brp:datumEinde>
                        <brp:buitenlandsePlaatsEinde>Barcelona</brp:buitenlandsePlaatsEinde>
                        <brp:buitenlandseRegioEinde>Catalonie</brp:buitenlandseRegioEinde>
                        <brp:landGebiedEindeCode>9096</brp:landGebiedEindeCode>
                    </brp:relatie>
                </brp:huwelijk>
            </brp:registratieEindeHuwelijk>
        </brp:acties>
    </brp:ontbindingHuwelijkInBuitenland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap>
