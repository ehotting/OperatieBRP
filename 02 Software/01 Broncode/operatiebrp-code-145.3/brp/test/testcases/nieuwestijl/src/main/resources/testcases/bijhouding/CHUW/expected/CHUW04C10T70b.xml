<brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>88409eeb-1aa5-43fc-8614-43055123a165</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Geslaagd</brp:verwerking>
        <brp:bijhouding>Verwerkt</brp:bijhouding>
        <brp:hoogsteMeldingsniveau>Geen</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:ontbindingHuwelijkInBuitenland brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
        <brp:bijgehoudenPersonen>
            <brp:persoon brp:objecttype="Persoon">
                <brp:identificatienummers>
                    <brp:burgerservicenummer>184306073</brp:burgerservicenummer>
                </brp:identificatienummers>
            </brp:persoon>
            <brp:persoon brp:objecttype="Persoon">
                <brp:identificatienummers>
                    <brp:burgerservicenummer>250020361</brp:burgerservicenummer>
                </brp:identificatienummers>
            </brp:persoon>
        </brp:bijgehoudenPersonen>
        <brp:bijhoudingsplan brp:objecttype="Bijhoudingsplan">*</brp:bijhoudingsplan>
    </brp:ontbindingHuwelijkInBuitenland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
